dev-tools/globals.ts

```diff
@@ -22,7 +22,7 @@ let TeamValidatorAsync = require('../team-validator-async');
 
 type GenderName = 'M' | 'F' | 'N' | '';
 type StatName = 'hp' | 'atk' | 'def' | 'spa' | 'spd' | 'spe';
-type StatsTable = {hp: number, atk: number, def: number, spa: number, spd: number, spe: number};
+type StatsTable = {hp: number, atk: number, def: number, spa: number, spd: number, spe: number, [stat: string]: number};
```

Comment:
Assuming this is to fix the build error from the ROA spotlight, I get that from our POV it cant crash, but making the type alot more open is not the answer. I don't know the fix myself as I spent some time trying earlier but haven't cracked it myself, but even a `// @ts-ignore` would be better than basically making this `type StatsTable {[stat: string]: number}`

Reply:
One option is to annotate the loop variable with `@type {StatName}`. But unfortunately I don't see how to do that without declaring it separately, which is even uglier than `@ts-ignore`.